### PR TITLE
 Enhance usage plugin

### PR DESCRIPTION
**Pull request by gj199575**
solve #3042 

First , Before this PR, if you are using usage plugin and there is pods is blocked by usage plugin because the node NodeUtilization is not normal , and you restart volcano-scheduler ,the usage plugin is not worked.
and in this PR ， I fix this little problem .

Second optimization usage plugin

### CODE REVIEW

File: pkg/scheduler/plugins/usage/usage.go [hunk 1]
```diff
@@ -43,35 +41,63 @@ const (
...
-          thresholds:
-            CPUUsageAvg.5m: 80
-            MEMUsageAvg.5m: 90
+         usage.weight: 10
+         type: average
+         thresholds:
+           cpu: 70
+           mem: 70
+         period: 10m
 */
```
Comment (lowang-bh) [inline pkg/scheduler/plugins/usage/usage.go:50]: if I want to take both Avg.10m and Avg.30m into account, how to config it?

File: pkg/scheduler/plugins/usage/usage.go [hunk 2]
```diff
@@ -99,26 +124,26 @@ func (up *usagePlugin) OnSessionOpen(ssn *framework.Session) {
...
+			case "mem":
+				up.memThresholds = float64(value)
 			}
-			if strings.Contains(key, memUsageAvgPrefix) {
-				periodKey := strings.Replace(key, memUsageAvgPrefix, "", 1)
-				up.threshold.memUsageAvg[periodKey] = val
+		}
+
+		// here is to deal with the case that restart volcano-scheduler and there is no ResourceUsage in cache of volcano-scheduler
+		if source.Period == "" {
```
Comment (lowang-bh) [inline pkg/scheduler/plugins/usage/usage.go:137]: why period == "" means the case volcano restarted?

File: pkg/scheduler/plugins/usage/usage.go [hunk 3]
```diff
@@ -99,26 +124,26 @@ func (up *usagePlugin) OnSessionOpen(ssn *framework.Session) {
...
+			source.Period = up.SamplePeriods
+			ssn.GetCache().GetMetricsData()
```
Comment (lowang-bh) [inline pkg/scheduler/plugins/usage/usage.go:139]: I don't think we need to fetch cache data from scheduler-cache. All the cached data is snapshoted in open session and we should use the snapshot data.

File: pkg/scheduler/plugins/usage/usage.go (hunk 1 above)
Comment (gj199575) [reply pkg/scheduler/plugins/usage/usage.go:50]: 1 firstly
I  think both use AVG and COMMON is not good,so I do not support it
2 secondly 
I  think both use 5m and 10m  is not good,so I do not support it either

by the way
I make a new PR
https://github.com/volcano-sh/volcano/pull/3056

and this PR is closed

File: pkg/scheduler/plugins/usage/usage.go (hunk 2 above)
Comment (gj199575) [reply pkg/scheduler/plugins/usage/usage.go:137]: yes,period == "" means   volcano  is just starting

File: pkg/scheduler/plugins/usage/usage.go (hunk 3 above)
Comment (gj199575) [reply pkg/scheduler/plugins/usage/usage.go:139]: when volcano is just starting, there is not   read use data in snapshot

File: pkg/scheduler/plugins/usage/usage.go (hunk 2 above)
Comment (lowang-bh) [reply pkg/scheduler/plugins/usage/usage.go:137]: why?

Comment (lowang-bh) [reply pkg/scheduler/plugins/usage/usage.go:137]: > why?

I know it now. period is a gloabal variable.

File: pkg/scheduler/plugins/usage/usage.go (hunk 1 above)
Comment (lowang-bh) [reply pkg/scheduler/plugins/usage/usage.go:50]: > I make a new PR

please update commit on the same PR. thanks.

**Pull request closed without merging**